changelog/changes/2022-08-03-coreos-cloudinit.md

```diff
@@ -0,0 +1 @@
+- coreos-cloudinit now supports IMDSv2 on AWS ([coreos-cloudinit#13](https://github.com/flatcar-linux/coreos-cloudinit/pull/13))
```

Comment:
```suggestion
- Added AWS IMDSv2 coreos-cloudinit support ([flatcar-linux/coreos-cloudinit#13](https://github.com/flatcar-linux/coreos-cloudinit/pull/13))
```

Reply:
I reworded it to more or less that now (Added AWS IMDSv2 support to coreos-cloudinit)